data/authors.yaml

```
@@ -1,9 +1,13 @@
 authors:
-    - id: marek-zaluski
+    - slug: marek-zaluski
       name: Marek Zaluski
       bio: Marek is a developer education engineer at Sourcegraph.
       socialLinks:
         - https://twitter.com/marekweb
         - https://www.linkedin.com/in/marekzaluski
-    - id: lisa-tagliaferri
+    - slug: marek-zaluski
```

I just spotted this issue - maybe a copy-paste error. This is causing Lisa's author page to be a 404. That could be an automated check: to make sure that there are no duplicate slugs (an idea for the future).
```suggestion
    - slug: lisa-tagliaferri
```